lib/src/widgets/code_block_component_widget.dart

```diff
@@ -15,10 +16,12 @@ class CodeBlockComponentWidget extends BlockComponentStatefulWidget {
     super.configuration = const BlockComponentConfiguration(),
     this.padding = const EdgeInsets.all(0),
     required this.editorState,
+    this.theme = builtInCodeBlockTheme,
```

Comment:
Allow the user to override the code block theme when they use this widget. AppFlowy will likely use fields from their `ThemeData`'s `ColorScheme` here.